package.json

```diff
@@ -26,6 +26,7 @@
     "lint": "run-p lint:*",
     "lint:es": "eslint . --cache -f friendly",
     "lint:tsc": "tsc --noEmit",
+    "prepack": "yarn build:ts",
```

Comment:
```suggestion
```

Not needed

https://github.com/rx-ts/changesets-gitlab/pull/23#issuecomment-974830529

Reply:
You are right, I removed it.